<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemFrame</class>
 <widget class="QWidget" name="itemFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>46</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Shopping List</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="itemLayout">
     <item>
      <widget class="QCheckBox" name="haveFoundCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numberOfItems">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Write number of units</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="itemName">
       <property name="toolTip">
        <string>Write item</string>
       </property>
       <property name="text">
        <string extracomment="Item"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="deleteButton">
       <property name="standardButtons">
        <set>QDialogButtonBox::Discard</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>itemName</tabstop>
  <tabstop>numberOfItems</tabstop>
  <tabstop>haveFoundCheckBox</tabstop>
  <tabstop>deleteButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>itemName</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>numberOfItems</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newItem_clicked()</slot>
 </slots>
</ui>
